src/quo/components/cards/wallet_card/view.cljs

```diff
@@ -0,0 +1,56 @@
+(ns quo.components.cards.wallet-card.view
+  (:require [quo.components.cards.wallet-card.style :as style]
+            [quo.components.icon :as icon]
+            [quo.components.markdown.text :as text]
+            [quo.theme :as quo.theme]
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]
+            [schema.core :as schema]))
+
+(def ^:private ?schema
```

Comment:
Malli schema should be in `schema.cljs` file like other components.

Reply:
Since it's a very small schema. I just placed it within the view. I will move it to `schema.cljs` 👍

Reply:
I understand but there are other smaller components too, so its best to keep the the convention.